corehq/apps/ota/views.py

```diff
@@ -74,9 +74,17 @@ def search(request, domain):
 @json_error
 @login_or_digest_or_basic_or_apikey()
 def claim(request, domain):
+    """
+    Allows a user to claim a case that they don't own.
+    """
+    # import ipdb; ipdb.set_trace()
```

Comment:
:flushed: I used to have a pre-commit hook to catch that. I wonder what happened to it.